.changeset/perfect-drinks-fold.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
+---
+
+Update the help output to improve formatting
```

Comment:
These changelogs can be combined

Reply:
They're both individually different features impacting different packages (for example, `create-astro` didn't receive a change to "update the help output"). 

I don't feel too strongly, but given that this is more correct I'd argue to keep as-is.